Cargo.toml

```diff
@@ -2,6 +2,156 @@
+sp-std = { git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+sp-timestamp = { git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+sp-tracing = { git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+sp-transaction-pool = { git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+sp-version = { git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+substrate-frame-rpc-system = { git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+substrate-wasm-builder = { git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+try-runtime-cli = { git = "https://github.com/humanode-network/substrate", branch = "locked/polkadot-v0.9.38" }
+
+# Frontier fork.
+fc-cli = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fc-consensus = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fc-db = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fc-mapping-sync = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fc-rpc = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fc-rpc-core = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fc-storage = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fp-evm = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fp-rpc = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fp-self-contained = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+fp-storage = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+pallet-base-fee = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+pallet-dynamic-fee = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+pallet-ethereum = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+pallet-evm = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+pallet-evm-precompile-modexp = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+pallet-evm-precompile-sha3fips = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+pallet-evm-precompile-simple = { git = "https://github.com/humanode-network/frontier", branch = "locked/polkadot-v0.9.38" }
+
```

Comment:
What is the difference including our crates as a part of workspace as well (like author-ext-api,...)? I've seen a lot of projects that include own crates into workspace description as well.

Reply:
We can do that too, it is just easier to distinguish between the two groups this way. I don't think it has any meaningful difference.